src/protections/WordList.ts

```diff
@@ -89,19 +83,25 @@ export class WordList extends Protection {
                     return
                 }
             }
-
+            if (!Boolean(this.badWords)) {
```

Comment:
```suggestion
            if (this.badWords == null) {
```

this catches `null` and `undefined`

Reply:
Surely you'd need to use strict equality to prevent that?